### ISSUE TITLE
A11y Issue: Wrong Control Receives Focus

### ISSUE THREAD

**Issue by carltonsmith** [raft-tech/TANF-app#407]
# Bug 1 

## Description 
- System error banners don't grab focus. 

## screenshot(s) of  expected and current behavior, 
![](https://i.imgur.com/PyzHDSM.png)

## steps to reproduce 
- Go to authenticated TDP
- Turn wifi off on device
- Activate request access button or other link

## Expected Behavior
- After a discussion with Robert, the expected behavior is that the focus remains on the `Submit Request` button but the alert is read out to anyone using a screen reader.

Note: I assume this is something baked into a USWDS component given that the same issue was present with the same type of banner while the error 403 bug was happening? Generate banner on x console error?

## severity
- Current state fails to meet WCAG/508 standards as any error served by this component would not be read by a screenreader when it occurred. 

## potential security implications where appropriate

**Comment by reitermb** [comment]
Noting here as well as the PR that this passed Raft Review in terms of a11y
